transports/janus_rabbitmq.c

```diff
@@ -684,6 +521,201 @@ int janus_rabbitmq_init(janus_transport_callbacks *callback, const char *config_
 	return -1;
 }
 
+int janus_rabbitmq_connect(void) {
+	rmq_client->connected = 0;
+	/* Connect */
+	rmq_client->rmq_conn = amqp_new_connection();
+	amqp_socket_t *socket = NULL;
+	amqp_queue_declare_ok_t *declare = NULL;
+	int status;
+	JANUS_LOG(LOG_VERB, "Creating RabbitMQ socket...\n");
+	if (ssl_enabled) {
+		socket = amqp_ssl_socket_new(rmq_client->rmq_conn);
+		if(socket == NULL) {
+			JANUS_LOG(LOG_FATAL, "Can't connect to RabbitMQ server: error creating socket...\n");
+			return -1;
+		}
+		if(ssl_verify_peer) {
+			amqp_ssl_socket_set_verify_peer(socket, 1);
+		} else {
+			amqp_ssl_socket_set_verify_peer(socket, 0);
+		}
+		if(ssl_verify_hostname) {
+			amqp_ssl_socket_set_verify_hostname(socket, 1);
+		} else {
+			amqp_ssl_socket_set_verify_hostname(socket, 0);
+		}
```

Comment:
Minor style point, these could be simplified to just:
```
amqp_ssl_socket_set_verify_peer(socket, ssl_verify_peer);
amqp_ssl_socket_set_verify_peer(socket, ssl_verify_hostname);
```
as gbooleans are just macros for 0 and 1:
https://developer.gnome.org/glib/stable/glib-Standard-Macros.html#TRUE:CAPS